sql/src/main/java/io/crate/analyze/relations/FullQualifedNameFieldProvider.java

```diff
@@ -57,15 +59,18 @@ public Field resolveField(QualifiedName qualifiedName, @Nullable List<String> pa
         String columnSchema = null;
         String columnTableName = null;
         ColumnIdent columnIdent = new ColumnIdent(parts.get(parts.size() - 1), path);
+        QualifiedName tableAndSchema = null;
         switch (parts.size()) {
             case 1:
                 break;
             case 2:
                 columnTableName = parts.get(0).toLowerCase(Locale.ENGLISH);
+                tableAndSchema = new QualifiedName(columnTableName);
                 break;
             case 3:
                 columnSchema = parts.get(0).toLowerCase(Locale.ENGLISH);
                 columnTableName = parts.get(1).toLowerCase(Locale.ENGLISH);
+                tableAndSchema = new QualifiedName(Arrays.asList(columnSchema, columnTableName));
```

Comment:
the tableAndSchema is only used in error cases, can you do the qualified name construction in the constructor